certbot/plugins/enhancement_test.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for new style enhancements"""
```

Comment:
We should rename this file to the plural "enhancements" to match the name of the module.